P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -2608,6 +2608,33 @@ effective than the free form <gi>note</gi> element; these elements are
 available when the module for simple analysis is selected (see section
 <ptr target="#AISP"/>).
  </p>
+
+ <p>When alternative versions of the same note, e.g. in different languages or
+   presenting a short and more elaborate form of a note it is possible to group
+   such notes within the <gi>noteGrp</gi> element. Such notes would make use of
+   attributes such as <att>xml:lang</att> or <att>type</att> to differentiate their
+   field of application while sharing the same point of attachment (either implicit
+   in case of inline notes or explicitly specified in a <att>target</att> attribute.
+  </p>
+
+  <p>Simple example below demonstrates grouping of shortened and full version of
+    the same note, perhaps shorter meant for media with space constraints.
```

Comment:
...where the shorter version might be intended for use in contexts with space constraints.